src/main/java/org/testng/TestNG.java

```diff
@@ -1044,7 +1045,31 @@ public void run() {
     runExecutionListeners(true /* start */);
 
     m_start = System.currentTimeMillis();
-    suiteRunners = runSuites();
+
+    //
+    // Slave mode
+    //
+    if (m_slavefileName != null) {
```

Comment:
the remote package was moved to https://github.com/testng-team/testng-remote in PR #953 
line 1048 ~ 1072 was removed, see PR #953